apps/module-scan/src/interpreter.ts

```diff
@@ -215,11 +216,15 @@ export default class Interpreter {
   private testMode: boolean
   private markThresholds: MarkThresholds
 
-  public constructor(election: Election, testMode: boolean) {
+  public constructor(
+    election: Election,
+    testMode: boolean,
+    markThresholdOverrides: Optional<MarkThresholds>
```

Comment:
should this be an object for named parameters at this point?

Reply:
I agree with this.